test/integration/guest_user_test.rb

```diff
@@ -43,6 +43,25 @@ class GuestUserTest < ActionDispatch::IntegrationTest
     assert page.has_content?(project.categories.first.name)
   end
 
+  test "a guest can visit the about page in the footer on any page" do
+    user = create(:user)
```

Comment:
This test is for a user visiting the about page, not a guest user as the test description asserts.